2/README.md

```diff
@@ -0,0 +1,122 @@
+```
+shortname: 2/COSS
+name: Consensus-Oriented Specification System
+status: raw
+editor: Alberto Granzotto <alberto@bigchaindb.com>
+```
+
+This document describes a consensus-oriented specification system (COSS) for building interoperable technical specifications. COSS is based on a lightweight editorial process that seeks to engage the widest possible range of interested parties and move rapidly to consensus through working code.
+
+This specification is based on the original [Digistan 1/COSS](http://www.digistan.org/spec:1/COSS)
+
+
+## Change Process
+This document is governed by the [2/COSS](../2/README.md) (COSS).
+
+## Language
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](http://tools.ietf.org/html/rfc2119)
+
+## Goals
+The primary goal of COSS is to facilitate the process of writing, proving, and improving new technical specifications. A "technical specification" defines a protocol, a process, an API, a use of language, a methodology, or any other aspect of a technical environment that can usefully be documented for the purposes of technical or social interoperability.
+
+COSS is intended to above all be economical and rapid, so that it is useful to small teams with little time to spend on more formal processes.
+
+Principles:
+
+* We aim for rough consensus and running code.
+* Specifications are small pieces, made by small teams.
+* Specifications should have a clearly responsible editor.
+* The process should be visible, objective, and accessible to anyone.
+* The process should clearly separate experiments from solutions.
+* The process should allow deprecation of old specifications.
+
+Specifications should take minutes to explain, hours to design, days to write, weeks to prove, months to become mature, and years to replace.
+
+Specifications have no special status except that accorded by the community.
+
+## Architecture
+COSS is designed around fast, easy to use communications tools. Primarily, COSS uses a wiki model for editing and publishing specifications texts.
```

Comment:
I'm confused by *wiki model*. Is keeping proposals under a GitHub repo a *wiki model*?

Reply:
I was also confused. My interpretation: maybe they mean the [gitbook](https://rfc.unprotocols.org/spec:2/COSS/)

Reply:
Maybe they mean a model where "anyone can edit," like in a wiki?